pkg/config/manifest.go

```diff
@@ -109,6 +110,26 @@ type Dependency struct {
 	Connections []BundleConnection `yaml:"connections",omitempty`
 }
 
+// OutputDefinition defines a single output for a CNAB
+type OutputDefinition struct {
+	Name        string   `yaml:"name"`
+	ApplyTo     []string `yaml:"applyTo,omitempty"`
+	Description string   `yaml:"description,omitempty"`
+	Sensitive   bool     `yaml:"sensitive"`
+
+	Schema `yaml:",inline"`
+}
+
+func (od *OutputDefinition) Validate() error {
+	if od.Name == "" {
+		return errors.New("output name is required")
+	}
+
+	// TODO: Validate inline Schema
```

Comment:
Haha, yeah that's a big task on our plate in general for CNAB 1.0 no need to tackle it as part of this PR. 😀